conf/nomad/trino_standalone.hcl

```diff
@@ -163,15 +180,19 @@ job "${nomad_job_name}" {
         image = "${docker_image}"
 %{ endif }
         volumes = [
-          "local/trino/config.properties:/lib/trino/default/etc/config.properties",
-          "local/trino/catalog/hive.properties:/lib/trino/default/etc/catalog/hive.properties",
+          "local/trino/config.properties:/etc/trino/config.properties",
+          "local/trino/catalog/hive.properties:/etc/trino/catalog/hive.properties",
+          # Trino extra config volume destination
+          "local/trino/catalog/postgresql.properties:/etc/trino/catalog/postgresql.properties",
           # JVM settings. Memory GC etc.
-          "local/trino/jvm.config:/lib/trino/default/etc/jvm.config",
+          "local/trino/jvm.config:/etc/trino/jvm.config",
```

Comment:
@claesgill Why did you change `lib/` to `/etc`?

Reply:
The old path did not work and Alexander enlightened me that the correct path should be `/etc/trino/` since `/lib/trino/default/etc/` is just a path "we" created. Putting any config in the old path does not seem to have any effect.